### ISSUE TITLE
missing rule: require operator before a line break

### ISSUE THREAD

**Issue by remitbri** [eslint/eslint#1620]
Hi. I've tried to review each and every rule of JSHint, JSCS and ESLint to see if I could entirely "get rid of" the first two in favour of the latter one. As suggested by the owner of the @geteslint twitter account, I'm reporting the JSCS rules I've found actually not covered by ESLint. My apologies if it's a duplicate.

 - "requireOperatorBeforeLineBreak"

…so that, yeah, the operator should not be after the line break. E.g:

```js
a = b
  ? c // invalid
  : d // invalid
  + 5 // invalid
if (b
  === a // invalid
  ) {b = 1}
```

**Comment by xjamundx** [comment]
https://github.com/jscs-dev/node-jscs/blob/master/lib/rules/require-operator-before-line-break.js

**Comment by lo1tuma** [comment]
Seems like a duplicate of #1405.

**Comment by nzakas** [comment]
Good call, let's track on #1405.
